dashboard/app/helpers/levels_helper.rb

```diff
@@ -137,23 +127,25 @@ def boolean_string_false
     "false"
   end
 
-  # Code for generating the blockly options hash
-  def blockly_options(local_assigns={})
+  # Generate a level-specific blockly options hash
+  def blockly_level_options(level)
```

Comment:
Now that this method doesn't reference any `@instance` vars it's a great candidate to move to the `Level` model!